### PR TITLE
Add more tests to blitFramebuffer for feedback loop: src buffer and dst buffer has the same image

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
Add more test cases to do a thorough test for this feature. 

PTAL. Thanks a lot.

### CODE REVIEW

File: sdk/tests/conformance2/rendering/blitframebuffer-test.html [hunk 1]
```diff
@@ -145,9 +154,51 @@
...
+        testFailed("Framebuffer incomplete.");
+        return;
+    }
+    // But the mask doesn't have depth buffer bit.
+    gl.blitFramebuffer(0, 0, 2, 2, 0, 0, 2, 2, gl.COLOR_BUFFER_BIT, gl.NEAREST);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "blitFramebuffer should succeed.");
+
+    // The mask has depth buffer bit.
+    gl.blitFramebuffer(0, 0, 2, 2, 0, 0, 2, 2, gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT, gl.NEAREST);
+    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, "blitFramebuffer should generate INVALID_OPERATION if read/draw framebuffer are identical.");
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/blitframebuffer-test.html]: Should the error message read "...if read/draw framebuffers have identical depth buffer attachments"?

File: sdk/tests/conformance2/rendering/blitframebuffer-test.html [hunk 2]
```diff
@@ -145,9 +154,51 @@
...
+        testFailed("Framebuffer incomplete.");
+        return;
+    }
+    // But the mask doesn't have stencil buffer bit.
+    gl.blitFramebuffer(0, 0, 2, 2, 0, 0, 2, 2, gl.COLOR_BUFFER_BIT, gl.NEAREST);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "blitFramebuffer should succeed.");
+
+    // The mask has stencil buffer bit.
+    gl.blitFramebuffer(0, 0, 2, 2, 0, 0, 2, 2, gl.COLOR_BUFFER_BIT | gl.STENCIL_BUFFER_BIT, gl.NEAREST);
+    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, "blitFramebuffer should generate INVALID_OPERATION if read/draw buffer are identical.");
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/blitframebuffer-test.html]: Make the comment more precise.

**Comment by kenrussell** [comment]
Thanks @Richard-Yunchao , this looks good. Could you make the small changes requested?

@zhenyao any additional feedback?

**Comment by zhenyao** [comment]
lgtm once kbr's comments are addressed.

**Comment by Richard-Yunchao** [comment]
Thanks for your review, @kenrussell  and @zhenyao . Code has been updated accordingly, please take another look. Thanks a lot!

**Comment by zhenyao** [comment]
lgtm
Thank you

**Comment by kenrussell** [comment]
Thanks. LGTM after the fact.

**Pull request merged**